pull

```diff
@@ -86,241 +52,147 @@ while [ "$1" != "" ]; do
+    # Extract fileadmin from docker container
+    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
+        cd $serverPath/ && mkdir -p tmp
+        docker cp main-application-1:/var/www/html/$webDirectory/fileadmin/ tmp/
+    "
+    catch "Could not extract fileadmin."
+
+    # Download fileadmin
+    rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --exclude="_processed_" --exclude="_temp_" --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include="*.youtube" --include='*/' --exclude="*" $serverAddressAndPath/tmp/fileadmin/ $webDirectory/fileadmin/
+    catch "Could not download fileadmin."
+    color green "Successfully downloaded fileadmin."
 fi
 if [ $software = 'statamic' ];
 then
-  rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include='*/' --exclude="*" $serverAddressAndPath/public/assets/ public/assets/
-fi
-if [ $software = 'shopware6' ];
-then
-  rsync -chavzP --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include="*.webp" --include='*/' --exclude="*" $serverAddressAndPath/public/media/ public/media/
-fi
-echo ''
+    # Extract assets from docker container
+    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
+        cd $serverPath/ && mkdir -p tmp
+        docker cp main-application-1:/var/www/html/public/assets/ tmp/
+    "
+    catch "Could not extract assets."
+    # Download assets
+    rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include='*/' --exclude="*" $serverAddressAndPath/tmp/assets/ public/assets/
+    catch "Could not download assets."
+    color green "Successfully downloaded assets."
 
-# Synchronize texts
-if [ $software = 'statamic' ];
-then
-  echo 'Synchronize texts'
-  echo '-----------------'
-  rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats $serverAddressAndPath/content/ content/
-fi
-echo ''
+    # Extract statamics static files from docker container
+    headline '📝 Synchronize texts'
+    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
+        cd $serverPath/ && mkdir -p tmp
```

Comment:
I didn’t test it, but the `tmp` folder might already exist here, containing the assets? If that’s the case, it should be deleted before

Reply:
Ah, sorry, I thought the complete content of the `tmp` folder is downloaded via `rsync`, that’s why I wrote the comment, I thought we have assets and content in `tmp` then and download things twice 🤦🏻‍♂️